robot-core/src/main/java/org/obolibrary/robot/ReportOperation.java

```diff
@@ -893,9 +908,9 @@ private static String getQueryResultOrNull(QuerySolution qs, String var) {
    * @return list of Violation objects
    * @throws Exception on malformed query
    */
-  private static List<Violation> getViolationsFromResults(
+  private static Set<Violation> getViolationsFromResults(
```

Comment:
You said you tried Set here to fix the diffs, but that didn't work. Do you really want Set now or is List better?

Reply:
I don't care either way, I can switch it back to lists if you prefer?

Reply:
Yes please.

Reply:
Done!